chainbase/src/main/java/org/tron/core/store/DynamicPropertiesStore.java

```diff
@@ -2523,6 +2525,10 @@ public boolean useNewRewardAlgorithm() {
     return getNewRewardAlgorithmEffectiveCycle() != Long.MAX_VALUE;
   }
 
+  public boolean useNewRewardAlgorithmFromStart() {
+    return getNewRewardAlgorithmEffectiveCycle() == 1;
```

Comment:
Can this value possibly equal 1, even on a new chain?

Reply:
Yes, It can be turned on anytime.